### ISSUE TITLE
"Select all" chapter option?

### ISSUE THREAD

**Issue by Greybane** [blacker-cz/MangaScraper#1]
It gets tiresome having to click every chapter on some of the long ones.

**Comment by blacker-cz** [comment]
Good point, I was actually thinking about adding this in the past. But right now there exists a workaround which you can use. 
If you go to the Settings tab, there is a 'Chapters selection mode:' which is by default set to 'Multiple' (that sticky selection, let's say). When you change it to 'Extended', the selection should work in a same way as you are used from a different application (only one item selected on click, multiple with ctrl+click or shift+click, with ctrl+a working as it should) and you should be able to select all the chapters without any difficulty. Let me know if that works for you.
Either way I'm gonna keep this issue open for now, so that it may get considered in one of the future versions (if there will be any).
